examples/web_testing/defaults.bzl

```diff
@@ -12,40 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Overrides for karma_web_test_suite & ts_web_test_suite to support bazelci testing
+"""Overrides for karma_web_test_suite & karma_web_test_suite to support bazelci testing
```

Comment:
needs some cleanup of places where it's just duplicating like this

Reply:
nice catch. just the one that I could see.